src/views/charts/PnlChart.tsx

```diff
@@ -168,9 +141,37 @@ export const PnlChart = ({
                 } as PnlDatum)
             )
         : [],
-    [pnlData, equity, selectedPeriod, now]
+    [pnlData, equity?.current, now]
   );
 
+  const getPeriodOptions = (oldestPnlMs: number): HistoricalPnlPeriods[] => {
+    const availablePeriods: HistoricalPnlPeriods[] = [HistoricalPnlPeriod.Period1d];
+
+    _.forEach(Object.entries(MS_FOR_PERIOD), ([, ms], i, entries) => {
```

Comment:
[q] why lodash `forEach` over js native Array `map`?

Reply:
haha no particular reason other than easy way to get the `entries`; but i can change it to .map instead

Reply:
edit: updated to reduce